README.md

```diff
@@ -32,22 +33,237 @@ scheduled_task { 'Run Notepad':
 
 ## Usage
 
-See the [Puppet resource documentation](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) for more information.
+One common use is to kick off a script, either once or on a regular cadence.
+In this first example we schedule a cleanup script to run this one time.
+
+~~~ puppet
+scheduled_task { 'Disk Cleanup': # Unique name for the scheduled task
```

Comment:
Not sure how I feel about inline comments.  IMO you show the example and then explain each line later.
If we are leaving inline comments, perhaps align the hash characers?

Reply:
I pulled the idea for inline comments from the existing examples on the v1 docs, I'm not married to them. The upside to inline comments is that they're really useful for at-a-glance reference and don't impede the copy-paste usage.

A major downside is they add clutter. I'll at least ensure the hashes align, missed that.

I _think_ most of the stuff covered in the inline comments is covered in the text, either here or in the reference section.

Reply:
FWIW, I usually like inline comments... but that's probably a style preference.